### PR TITLE
Prevent redeploy pre existing workers unless specified

### PR DESCRIPTION
**Pull request by serngawy**
Signed-off-by: melserngawy <melserng@redhat.com>

### CODE REVIEW

File: ai-deploy-cluster-remoteworker/README.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ and Small ISO.
 
     - You can download the kubeconfig of your cluster using the command below
 
-            $ aicli download installconfig {YOUR_CLUSTER_NAME} --path ~/
+            $ aicli download kubeconfig {YOUR_CLUSTER_NAME} --path ~/
```
Comment (yrobla) [inline ai-deploy-cluster-remoteworker/README.md:124]: this doesn't seem to be related

File: ai-deploy-cluster-remoteworker/inventory/hosts.sample [hunk 2]
```diff
@@ -37,5 +37,5 @@ need_racadm=true
 localhost ansible_connection=local
 
 [worker_nodes]
-worker-0.test-aut.cluster-testing bmc_type=SuperMicro bmc_address=192.168.111.212 bmc_user="ADMIN" bmc_password="ADMIN" smb_host=192.168.111.1 smb_path=share smb_server_path=/home/share ramdisk_path=/opt/network-config kernel_arguments="" final_iso_path=/home/share/ ignition_name=worker0
-worker-1 bmc_type=Dell bmc_address=10.16.231.121 bmc_user="usr" bmc_password="pwd" ramdisk_path=/opt/network-config kernel_arguments="" final_iso_path=/opt/cached_disconnected_images ignition_name=worker1
+worker-0.test-aut.cluster-testing bmc_type=SuperMicro bmc_address=192.168.111.212 bmc_user="ADMIN" bmc_password="ADMIN" smb_host=192.168.111.1 smb_path=share smb_server_path=/home/share ramdisk_path=/opt/network-config kernel_arguments="" final_iso_path=/home/share/ ignition_name=worker0 redploy=false
```
Comment (yrobla) [inline ai-deploy-cluster-remoteworker/inventory/hosts.sample:40]: typo, redeploy

**Review by yrobla — CHANGES_REQUESTED** [review]

File: ai-deploy-cluster-remoteworker/inventory/hosts.sample (hunk 2 above)
Comment (serngawy) [reply ai-deploy-cluster-remoteworker/inventory/hosts.sample]: done

**Review by yrobla — APPROVED** [review]
